README.md

```diff
@@ -149,3 +149,195 @@ client.datasources.drop('my_datasource')
 ```
 >Note: The SDK currently does not support automatically removing a data source if it is no longer connected to any mind.
 
+
+
+
+
+####Other SDK
+
+Here is the formatted content for `README.md`:
```

Comment:
We need to remove this code as it was added directly on the file.

Reply:
ok sir